internal/oci/container.go

```diff
@@ -695,3 +696,55 @@ func (c *Container) RestoreIsOCIImage() bool {
 func (c *Container) SetRestoreIsOCIImage(restoreIsOCIImage bool) {
 	c.restoreIsOCIImage = restoreIsOCIImage
 }
+
+// SetResources loads the OCI Spec.Linux.Resources in the container struct
```

Comment:
just had this thought: what if we combined this with `SetSpec`? then we wouldn't have to remember to call each time SetSpec was called

Reply:
makes sense

Reply:
Done